csep/utils/readers.py

```diff
@@ -756,4 +756,33 @@ def load_quadtree_forecast(ascii_fname):
     # reshape rates into correct 2d format
     rates = data[:, -1].reshape(n_poly, n_mag_bins)
 
+    return rates, region, mws
+
+
+def load_quadtree_forecast_csv(csv_fname):
```

Comment:
I think we change this name to `quadtree_csv_loader`. Reason: this function name implies that it will return a forecast object. Also should change the name of the other function to `quadtree_ascii_loader`.